src/iproute2mac.py

```diff
@@ -12,14 +12,32 @@
 
 import json
 import random
-import re
 import subprocess
 import sys
 import types
 
 # Version
 VERSION = "1.5.4"
 
+# Colors
+class Colors:
+    RESET = "\033[0m"
+    RED = "\033[31m"
+    GREEN = "\033[32m"
+    YELLOW = "\033[33m"
+    BLUE = "\033[34m"
+    PURPLE = "\033[35m"
+    CYAN = "\033[36m"
```

Comment:
It may be better to use a library like https://pypi.org/project/colored/ ?

Reply:
Thank you for your suggestion, but I don't want to introduce any external packages into my code as it will add burden.

Reply:
If the owner is willing, I would be happy to add.